src/main/java/uk/gov/hmcts/cmc/claimstore/exceptions/DocumentManagementException.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.hmcts.cmc.claimstore.exceptions;
+
+public class DocumentManagementException extends Throwable {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename this class to remove "Exception" or correct its inheritance. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS2166)